projects/vdk-plugins/vdk-kerberos-auth/README.md

```diff
@@ -8,6 +8,17 @@ To install the plugin, run:
 pip install vdk-kerberos-auth
 ```
 
+After it's install what happens:
+
+1. Upon installation and if KEYTAB_FILENAME and KEYTAB_PRINCIPAL are configured, it will try to authenticate ("kinit") at the start of very VDK command.
```

Comment:
```suggestion
1. Upon installation and if KEYTAB_FILENAME and KEYTAB_PRINCIPAL are configured, it will try to authenticate ("kinit") at the start of every VDK command.
```